src/mad_spark_alt/cli.py

```
@@ -517,9 +517,7 @@ async def _run_evolution_pipeline(
             console.print(
                 f"[green]✅ Generated {len(initial_ideas)} initial ideas[/green]"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)